packages/imperative/src/config/src/ConfigBuilder.ts

```diff
@@ -30,31 +31,10 @@ export class ConfigBuilder {
         const config: IConfig = Config.empty();
 
         for (const profile of impConfig.profiles) {
-            const properties: { [key: string]: any } = {};
-            const secureProps: string[] = [];
-            for (const [k, v] of Object.entries(profile.schema.properties)) {
-                if (opts.populateProperties && v.includeInTemplate) {
-                    if (v.secure) {
-                        secureProps.push(k);
-                    } else {
-                        if (v.optionDefinition != null) {
-                            // Use default value of ICommandOptionDefinition if present
-                            properties[k] = v.optionDefinition.defaultValue;
-                        }
-                        if (properties[k] === undefined) {
-                            // Fall back to an empty value
-                            properties[k] = this.getDefaultValue(v.type);
-                        }
-                    }
-                }
-            }
+            const defaultProfile = this.buildDefaultProfile(profile, opts);
```

Comment:
Yes, good catch! I think I got lucky and avoided a compile error, because `this` would end up being a static `ConfigBuilder` 😅  but best to be explicit. Resolved in upcoming commit 👍